### PR TITLE
Feature/request section

### PR DESCRIPTION
**Pull request by RobsonOlv**
#### What is the purpose of this pull request?

To update and style the request section.

#### What problem is this solving?

Inadequate structure and styling to follow [the expected design](https://www.figma.com/file/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=2150%3A138907) for the section showing the request params(with default values).

#### How should this be manually tested?

Check if there are any typos or styling errors in the added code. Following the new pattern to testing-version tags, you can view the achieved results on [this deploy preview](https://deploy-preview-44--elated-hoover-5c29bf.netlify.app/).

#### Screenshots or example usage
| Before | After |
|--------|-------|
| image|image|
#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: src/components/api-request.js [hunk 1]
```diff
@@ -331,149 +341,112 @@ export default class ApiRequest extends LitElement {
           false,
         )[0].exampleValue;
       }
-      const labelColWidth = 'read focused'.includes(this.renderStyle) ? '200px' : '160px';
+      // const labelColWidth = 'read focused'.includes(this.renderStyle) ? '200px' : '160px';
```
Comment (amandascm) [inline src/components/api-request.js]: Maybe this line could be removed to avoid unnecessary comments

**Review by amandascm — COMMENTED** [review]
Very nice!! ✨ 
Just noticed the parameters types are placed above the param names, while the expected is that it remains next to the 'required' label.
| Expected | Implemented |
|--------|-------|
| image|image|

**Review by leonardoAnjos16 — APPROVED** [review]
LGTM 💯

**Review by JosueFidelis — APPROVED** [review]
LGTM :top:

File: src/components/api-request.js (hunk 1 above)
Comment (RobsonOlv) [reply src/components/api-request.js]: I will change that

**Review by melissafalcao — APPROVED** [review]

**Review by amandascm — APPROVED** [review]
LGTM 💯

**Pull request merged**